src/netex_idf/lines.rs

```diff
@@ -95,12 +99,19 @@ fn load_netex_lines(
 fn make_lines(lines_netex_idf: &CollectionWithId<LineNetexIDF>) -> Result<CollectionWithId<Line>> {
     let mut lines = CollectionWithId::default();
     for ln in lines_netex_idf.values() {
+        let commercial_mode_id = skip_fail!(MODES
+            .get::<str>(&ln.mode)
+            .map(|m| {
+                let (cid, _) = m.commercial_mode;
+                cid.to_string()
```

Comment:
Or you could do in one line with
```suggestion
                m.commercial_mode.0.to_string()
```